.claude/skills/add-uielement/SKILL.md

```
@@ -0,0 +1,206 @@
+import { z } from "zod";
+import { BaseBoxPropsSchema, type BaseBoxProps } from "./BaseBoxProps";
+
+export type {ElementName}ElementProps = BaseBoxProps & {
+  // element-specific props here
+};
+
+export const {ElementName}ElementPropsSchema = BaseBoxPropsSchema.extend({
+  // zod schema for props here
+});
+```
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)